src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/fingerprints/BottomNavigationBarFingerprint.kt

```diff
@@ -1,18 +1,24 @@
 package app.revanced.patches.youtube.layout.hide.shorts.fingerprints
 
+import app.revanced.patcher.extensions.or
 import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
 import com.android.tools.smali.dexlib2.Opcode
 
 object BottomNavigationBarFingerprint : MethodFingerprint(
+    returnType = "V",
+    accessFlags = AccessFlags.PUBLIC or AccessFlags.FINAL,
+    parameters = listOf("Landroid/view/View;", "Landroid/os/Bundle;"),
     opcodes = listOf(
-        Opcode.MOVE_RESULT_OBJECT, // Refers to bottom navigation bar
+        Opcode.CONST, // R.id.app_engagement_panel_wrapper
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT_OBJECT,
```

Comment:
I assume this breaks compatibility with old versions, so it is important to not forget dropping support for it